### PR TITLE
Update pricing.js

### PR DESCRIPTION
**Pull request by mjuwit**
Added 1 million inodes to storage space pricing, plus various edits.

### CODE REVIEW

File: src/pages/pricing.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ export default function Pricing() {
...
-            Therefore, the total cost considerations for compute nodes in HYAK can be broken down into the sum of the following two components.
+            <p>The HYAK clusters operate on a condo model. This means that the cluster is itself consists of contributed resource slices from various groups across campus. The HYAK team, funded through the office of research and sponsoring entities, provides the core infrastructure (e.g., networking, storage, support staff). This is why faculty that are from sponsoring entities do not have any annual, ongoing costs associated with their slices beyond the initial cost of the hardware. The leadership of their sponsoring entities cover this. Faculty that are <u>not</u> affiliated with sponsoring entities have to shoulder this annual, ongoing cost associated with any slices they wish to contribute.</p>
```
Comment (nhrmic) [inline src/pages/pricing.js]: In " cluster is itself consists of" word "is" seems unnecessary

File: src/pages/pricing.js [hunk 2]
```diff
@@ -26,9 +26,9 @@ export default function Pricing() {
...
+            <p>You get access to resources equivalent to the slices your account contributes to the cluster on-demand. A cluster account also provides you access to all the other contributed slices from other entities, subject to their availability (i.e., if the contributors or the resources aren't actively using them). This is referred to as the "checkpoint" partition due to the lack of job run-time guarantees. Once a checkpoint job starts it can be re-queued at any moment, but is not uncommon for a job to run for 4 to 5 hours before requeue. Longer checkpoint jobs will continue to run and be re-queued until it completes, which is why it is important that your job be able to checkpoint or save state and resume gracefully. Checkpoint access can provide access to substantial resources beyond what you contribute and is the benefit of joining a shared cluster like HYAK compared to buying the same hardware operating your own server.</p>
```
Comment (nhrmic) [inline src/pages/pricing.js]: "re-queued until it completes" refers to "jobs". I think "until they complete" should be used.

File: src/pages/pricing.js [hunk 3]
```diff
@@ -196,8 +196,8 @@ export default function Pricing() {
               GPU Slices:
               <ul>
               <li>All slices are standardized on AMD EPYC 9534 CPUs ("Genoa"). We are on the NVIDIA "Ada" and "Hopper" generation of GPUs.</li>
-                <li>4 x GPU slices constitutes a single physical server (or node). It is a single box with 128-cores, 1.5TB of memory, and 8 x GPUs of the same type. They are sold in resource slices to make this a more tractible cost for labs with more modest GPU needs.</li>
-                <li>Any jobs requiring more than 8 x GPUs of the same type should be prepared to make use of message passing libraries (e.g., PyTorch Lightning) to scale across multiple servers. Any job up to the equivalent of 4 x GPU nodes (i.e., 8 x GPU cards) can be run on the same physical machine and therefore scale easily without much further modification to the codebase.</li>
+                <li>4 x GPU slices constitute a single physical server (or node). It is a single box with 128-cores, 1.5TB of memory, and 8 x GPUs of the same type. They are sold in resource slices to make this a more tractible cost for labs with more modest GPU needs.</li>
```
Comment (nhrmic) [inline src/pages/pricing.js]: tractable

**Review by mwanek — APPROVED** [review]
Comments from Slack have been addressed; approved.

**Pull request merged**